apps/site/pages/docs/getting-started/getting-started-overview.mdx

```diff
@@ -17,18 +17,25 @@ This documentation is currently under development.
 
 In this Getting Started track, you'll build and deploy a FastStore Storefront at the domain `https://<account-name>.vtex.app/`.
 
-By the end of this track, you'll have a fully functional store using FastStore, seamlessly integrated with VTEX native solutions like [Headless CMS](/tbd), [Catalog](/tbd), and [Checkout](/tbd).
+By the end of this track, you'll have a fully functional store using FastStore, seamlessly integrated with VTEX native solutions like [Headless CMS]([/tbd](https://www.faststore.dev/tutorials/cms/0)).
```

Comment:
Is this `[/tbd]` intentional?

Reply:
No, thanks, Guila!🙏 I'm going to commit the fix for it